src/chunk-stream.iced

```diff
@@ -1,17 +1,27 @@
 stream = require('stream')
+{make_esc} = require('iced-error')
 
 exports.ChunkStream = class ChunkStream extends stream.Transform
 
-  constructor : ({@transform_func, @block_size, @exact_chunking, @writableObjectMode, @readableObjectMode}) ->
+  constructor : ({@transform_func, @block_size, @readableObjectMode}) ->
     @extra = null
-    super({@writableObjectMode, @readableObjectMode})
+    super({@readableObjectMode})
+
+  _transform_chunk : (chunk, cb) ->
```

Comment:
I think it's cleaner to get rid of this function and inline it. Normally I'm a big fan of breaking things up, but when the smaller functions have to remember intimate details about each other's implementations, it can hurt more than it helps. Note the implicit contract around `@extra` here. We need to know that it's empty on line 15, which depends on what's happening in the caller at lines 27-28.

Anyway, overall just a slight preference of mine. Your call.